Requirement already satisfied: python-dateutil in c:\users\gis\appdata\local\programs\python\python39\lib\site-packages (1.5)
